internal/provider/security/security_security_login_message_resource_test.go

```diff
@@ -0,0 +1,74 @@
+package security_test
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"testing"
+
+	ntest "github.com/netapp/terraform-provider-netapp-ontap/internal/provider"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccSecurityLoginMessage(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { ntest.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: ntest.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Creating security_login_messages test
+			{
+				Config:      testAccSecurityLoginMessageBasicClusterConfig(),
+				ExpectError: regexp.MustCompile("create operation is not supported"),
+			},
+			// Import and read with cluster
```

Comment:
Is there a test for Modify?

Reply:
The test cannot be done on imported resource here.